config/locales/devise.en.yml

```diff
@@ -1,84 +1,76 @@
-# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
-
+---
```

Comment:
Yes, this is an unfortunate known issue: https://github.com/glebm/i18n-tasks/issues/120
There were over 40 unused keys, and I didn't feel like spending the time to remove them one by one manually. The tests pass, so we know that we didn't remove anything that we need, and that we no longer have unused translations.

If you feel strongly about separating the removal from the normalization, I can fork the gem and stop it from automatically sorting.